### PR TITLE
feat: add `chsh` spec

### PR DESCRIPTION
**Pull request by mrhappyma**
https://man.archlinux.org/man/chsh.1.en

### CODE REVIEW

File: src/chsh.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const shells: Fig.Generator = {
```
Comment (fedeci) [inline src/chsh.ts:1]:
```suggestion
// TODO: this does not work on macos
const shells: Fig.Generator = {
```

**Review by fedeci — APPROVED** [review]
Thanks

File: src/chsh.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      description: "Print help message and exit",
+    },
+    {
+      name: ["-v", "--version"],
+      description: "Print version and exit",
+    },
+  ],
+  args: {
+    name: "username",
+    description: "Target user",
```
Comment (fedeci) [inline src/chsh.ts:40]:
```suggestion
    description: "Target user",
    isOptional: true,
```

Comment (mrhappyma) [reply src/chsh.ts:40]: good catch

**Pull request merged**